charts/nango/templates/jobs/jobs-deployment.yaml

```diff
@@ -15,8 +15,9 @@ spec:
     spec:
       containers:
         - name: {{ .Values.jobs.name | default "nango-jobs" }}
-          image: nangohq/nango-jobs:{{ .Values.jobs.tag | default "enterprise" }}
+          image: nangohq/nango:prod-{{ .Values.jobs.tag | default .Values.shared.tag }}
```

Comment:
Note: the new image hasn't been tested for this service (it should but...)

Reply:
anything preventing us to switch `jobs` in prod/staging to the unified image?

Reply:
time 
It's annoying because you can't update automatically, so we need to change the render deployment manually then merge the PR. Also I wanted to wait for jobs to be killable before doing that, in case I encounter an issue

Reply:
Will keep this as is but agree we should definitely test using the unified image.